cfme/fixtures/vm.py

```diff
@@ -100,20 +100,16 @@ def _get_vm_name(request):
 
 @pytest.fixture(scope="module")
 def create_vm_modscope(setup_provider_modscope, request, provider):
-    if request.param:
-        template_type = request.param
-    else:
-        pytest.error('Any appropriate Template was not passed to the fixture.')
+    # If no template parameter is passed then use DSL.
```

Comment:
`DSL` is not always the default `small_template`, maybe just
```suggestion
    # If no template parameter is passed, use 'small_template'.
```

Reply:
Okay Updating it.

Reply:
Updated.